libs/common/src/lib/hint/hint.component.ts

```diff
@@ -42,21 +35,26 @@ export class HintComponent implements AfterContentInit, AfterViewInit, OnChanges
   }
   private _noMaxWidth = false;
 
-  isExpanded = false;
-
   @ViewChild('content') content?: ElementRef;
   @ViewChild('container') container?: ElementRef;
+  @ViewChild('codeExample') codeExample?: ElementRef;
+  @ViewChild('instructions') instructions?: ElementRef;
```

Comment:
These two viewChild are used ?

Reply:
not anymore, I forgot to remove them. I'll clean up, thanks